From 8a21b8db7059bce37615e305379c64ebafb0aa98 Mon Sep 17 00:00:00 2001
From: Kuan-Jui Chiu <kchiu@axiado.com>
Date: Tue, 21 Jul 2026 23:48:51 -0700
Subject: [PATCH] Increase MAXIMUM_TRANSFER_SIZE of PLDM

Upstream-Status: Inappropriate [Axiado SoC adaptation]
Signed-off-by: Kuan-Jui Chiu <kchiu@axiado.com>
---
 fw-update/update_manager.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/fw-update/update_manager.cpp b/fw-update/update_manager.cpp
index 3cf5fde..6daed58 100644
--- a/fw-update/update_manager.cpp
+++ b/fw-update/update_manager.cpp
@@ -182,7 +182,7 @@ void UpdateManager::processStream(std::istream& package, uintmax_t packageSize)
     const auto& fwDeviceIDRecords = parser->getFwDeviceIDRecords();
     const auto& compImageInfos = parser->getComponentImageInfos();
 
-    static constexpr uint32_t MAXIMUM_TRANSFER_SIZE = 4096;
+    static constexpr uint32_t MAXIMUM_TRANSFER_SIZE = 1048567;
     for (const auto& deviceUpdaterInfo : deviceUpdaterInfos)
     {
         const auto& fwDeviceIDRecord =
-- 
2.34.1

